manifests/plugin.pp

```diff
@@ -150,13 +157,13 @@
 
     exec { "download-${name}" :
       command => "rm -rf ${name} ${name}.hpi ${name}.jpi && wget --no-check-certificate ${download_url}",
-      cwd     => $plugin_dir,
```

Comment:
This needs to be rebased on master as https://github.com/jenkinsci/puppet-jenkins/pull/335 removed usage of wget from `plugin.pp`.